movement/movement_config.h

```diff
@@ -28,6 +28,7 @@
 #include "movement_faces.h"
 
 const watch_face_t watch_faces[] = {
+    menstrual_cycle_face,
```

Comment:
This line should get removed — we don't want to change the default set of faces.